test/extensions/filters/http/lua/lua_integration_test.cc

```diff
@@ -387,5 +389,98 @@ name: envoy.lua
   cleanup();
 }
 
+// Basic test for verifying signature.
+TEST_P(LuaIntegrationTest, SignatureVerification) {
+  const std::string FILTER_AND_CODE =
+      R"EOF(
+name: envoy.lua
+config:
+  inline_code: |
+    function string.fromhex(str)
+      return (str:gsub('..', function (cc)
+        return string.char(tonumber(cc, 16))
+      end))
+    end
+
+    -- decoding
+    function dec(data)
```

Comment:
This is a gem, can we put it in the doc (as for an example of decoding)?

Reply:
This comes from http://lua-users.org/wiki/BaseSixtyFour